ext/hash/hash_ripemd.c

```diff
@@ -639,7 +639,7 @@ static void RIPEMDEncode(unsigned char *output, uint32_t *input, unsigned int le
  */
 PHP_HASH_API void PHP_RIPEMD128Final(unsigned char digest[16], PHP_RIPEMD128_CTX * context)
 {
-	unsigned char bits[8];
+	unsigned char bits[8] = {0};
```

Comment:
This looks unnecessary if all elements are initialized below. Can this be suppressed instead?

Reply:
I think I might just raise a bug report on the GCC project, as yeah those shouldn't be flagged anyway